src/components/Pagination.jsx

```diff
@@ -54,13 +57,12 @@ function Pagination({
         return (
           <li
             key={key}
-            className={classnames("paginationItem", {
-              selected: false, // change this logic to add selected class name
-            })}
+            className="paginationItem"
+            aria-current="false" // change this line to highlight a current page.
```

Comment:
Replaced `.selected` class with `aria-current` attribute.
Candidates are expected to do this:
```
aria-current={pageNumber === currentPage}
```